src/views/LimitedRestorationExtension.vue

```diff
@@ -292,7 +289,7 @@ export default class LimitedRestorationExtension extends Mixins(
       this.setResource(this.restorationResource)
 
       // initialize Fee Summary data
-      this.setFilingData([this.restorationResource.filingData as unknown as FilingDataIF])
+      this.setFilingData([this.restorationResource.filingData])
```

Comment:
This now works because filingData and setFilingData refer to the same local type. Sometimes VS Code complains when using the same shared type. I'm guess it's an issue with how VS Code looks for imported types, perhaps a "d.ts" file? This can be a future problem to solve. For now, the code works.